Commands/Lists/RemoveView.cs

```diff
@@ -9,15 +9,19 @@ namespace SharePointPnP.PowerShell.Commands.Lists
     [Cmdlet(VerbsCommon.Remove, "SPOView", SupportsShouldProcess = true)]
     [CmdletHelp("Deletes a view from a list",
         Category = CmdletHelpCategory.Lists)]
+    [CmdletExample(
+        Code = @"PS:> Remove-SPOView -List ""Demo List"" -Identity ""All Items"" -Force",
+        SortOrder = 1,
+        Remarks = @"Removes the view with title ""All Items"" from the ""Demo List"" list.")]
```

Comment:
and skips the confirmation question. Better maybe to leave the -Force parameter out of the example.,